kustomize/overlays/test/kustomization.yaml

```diff
@@ -19,23 +19,19 @@ bases:
 images:
   # FlyteAdmin
   - name: flyteadmin # match images with this name
-    newTag: v0.4.0 # override the tag
+    newTag: v0.4.0 # FLYTEADMIN_TAG override the tag
     newName: ghcr.io/flyteorg/flyteadmin # override the name
   # FlyteConsole
   - name: flyteconsole # match images with this name
-    newTag: v0.19.6 # override the tag
+    newTag: v0.19.7 # FLYTECONSOLE_TAG  the tag
     newName: ghcr.io/flyteorg/flyteconsole # override the namep
   # Flyte DataCatalog
   - name: datacatalog # match images with this name
-    newTag: v0.3.0 # override the tag
+    newTag: v0.3.1 # DATACATALOG_TAG override the tag
     newName: ghcr.io/flyteorg/datacatalog # override the name
   # FlytePropeller
   - name: flytepropeller # match images with this name
-    newTag: v0.7.8 # override the tag
-    newName: ghcr.io/flyteorg/flytepropeller # override the name
-  # Webhook
-  - name: webhook # match images with this name
-    newTag: v0.7.8 # override the tag
+    newTag: v0.7.8 # FLYTEPROPELLER_TAG override the tag
```

Comment:
See on the left? a new image is added (webhook) it uses the same flytepropeller image... looks like the master merge here didn't happen properly